mmv1/products/compute/api.yaml

```diff
@@ -9750,6 +9750,70 @@ objects:
                     - :NEVER
                     - :ON_PERMANENT_INSTANCE_DELETION
                   default_value: :NEVER
+          - !ruby/object:Api::Type::Map
+            name: 'internalIp'
+            api_name: internalIPs
+            min_version: beta
+            key_name: "interface_name"
+            description: |
+              Preserved internal IPs defined for this instance. This map is keyed with the name of the network interface.
+            value_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::Enum
+                  name: autoDelete
+                  description: |
+                    A value that prescribes what should happen to the stateful disk when the VM instance is deleted.
+                    The available options are `NEVER` and `ON_PERMANENT_INSTANCE_DELETION`.
+                    `NEVER` - detach the disk when the VM is deleted, but do not delete the disk.
+                    `ON_PERMANENT_INSTANCE_DELETION` will delete the stateful disk when the VM is permanently
```

Comment:
Address

Reply:
Thanks for pointing it out, I'll change the description to what's on the public API page.
https://cloud.google.com/compute/docs/reference/rest/beta/instanceGroupManagers/patchPerInstanceConfigs